cmd/controller/main.go

```diff
@@ -59,6 +59,11 @@ func main() {
 	var probeAddr string
 	var secureMetrics bool
 	var enableHTTP2 bool
+	// Zap logger flags
```

Comment:
Looking at Kueue and JobSet, I see that we can use [zap.Add](https://github.com/kubernetes-sigs/jobset/blob/b92cbdcbe77f168a82cf12c9f349bdd053d2ae5c/main.go#L84) as an example.